src/testing/h_refinement_study_isentropic_vortex.cpp

```diff
@@ -192,6 +232,11 @@ int HRefinementStudyIsentropicVortex<dim, nstate>::run_test() const
         }
 
         // Convergence for pressure
+        if (this->all_parameters->flux_reconstruction_type == PHiLiP::Parameters::AllParameters::Flux_Reconstruction::user_specified_value){
```

Comment:
The table output from this is confusing to read. Since successive `c` values are solved at 32x32 and then 64x64, only every second row is relevant. This should be improved before merging.